### PR TITLE
docs: add background doc and playground

### PR DESCRIPTION
**Pull request by gz65555**
Add background docs and playground for [background texture feature](https://github.com/oasis-engine/engine/pull/427)

### CODE REVIEW

**Review by eyworldwide — APPROVED** [review]
+1

File: docs/scene.zh-CN.md [hunk 1]
```diff
@@ -81,8 +81,27 @@ background.mode = BackgroundMode.Sky; // 默认纯色背景
...
 
+##### 纹理背景适配模式
+
+通过 `background.textureFillMode = BackgroundTextureFillMode.AspectFitWidth` 设置纹理适配模式。
+
+目前纹理适配模式有以下三种：
+
+| 适配模式        | 说明                                               |
+| --------------- | -------------------------------------------------- |
+| AspectFitWidth  | 保持宽高比，把纹理宽缩放至 Canvas 的宽，上下居中。 |
```
Comment (eyworldwide) [inline docs/scene.zh-CN.md:99]: Add api link

**Review by GuoLei1990 — APPROVED** [review]
+1

File: playground/background.ts [hunk 2]
```diff
@@ -26,9 +27,13 @@ cameraEntity.addComponent(OrbitControl);
...
 engine.run();
 
+engine.canvas._webCanvas.addEventListener("onresize", () => {
+  engine.canvas.resizeByClientSize();
+});
+
 engine.resourceManager
   //@ts-ignore
-  .load<TextureCubeMap[]>([
+  .load<[TextureCubeMap, TextureCubeMap, Texture2D]>([
```
Comment (GuoLei1990) [inline playground/background.ts:36]: Maybe this is a wrong grammar.

File: playground/background.ts [hunk 3]
```diff
@@ -50,21 +55,33 @@ engine.resourceManager
...
+    // background.mode = BackgroundMode.Sky; // 默认纯色背景
     const skyMaterial = (background.sky.material = new SkyBoxMaterial(engine)); // 添加天空盒材质
     skyMaterial.textureCubeMap = cubeMap1; // 设置立方体纹理
     background.sky.mesh = PrimitiveMesh.createCuboid(engine, 2, 2, 2); // 设置天空盒网格
+    background.mode = BackgroundMode.Texture;
+    background.texture = texture;
     return [cubeMap1, cubeMap2];
   })
   .then((cubeMaps) => {
     const gui = new dat.GUI();
```
Comment (GuoLei1990) [inline playground/background.ts:75]: Extract a `addGUI()` function like other sample

**Comment by GuoLei1990** [comment]
image
Should improve:
Hidden some item when use some mode.

File: playground/background.ts (hunk 2 above)
Comment (gz65555) [reply playground/background.ts:36]: It is not a wrong grammer. It is just a generic.

**Comment by gz65555** [comment]
> image
> Should improve:
> Hidden some item when use some mode.

ok, fixed already.

File: playground/background.ts (hunk 3 above)
Comment (gz65555) [reply playground/background.ts]: fixed already

File: docs/scene.zh-CN.md (hunk 1 above)
Comment (gz65555) [reply docs/scene.zh-CN.md]: fixed already.

**Pull request merged**